ProjetSupaero2018/catkin_ws/src/display/scripts/gazebo_display.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import rospy
+from rospy_tutorials.msg import Floats
```

Comment:
I used a Point msg for the car_position publisher, so that's what you'll receive:

```python
from geometry_msgs.msg import Point
```